staging/src/k8s.io/cli-runtime/pkg/resource/builder.go

```diff
@@ -1213,11 +1213,12 @@ func expandIfFilePattern(pattern string) ([]string, error) {
 	if _, err := os.Stat(pattern); os.IsNotExist(err) {
 		matches, err := filepath.Glob(pattern)
 		if err == nil && len(matches) == 0 {
-			return nil, fmt.Errorf("no match")
+			return nil, fmt.Errorf("the path %q does not exist", pattern)
```

Comment:
Do we need this to remain the *exact same* as the error returned when a literal path is used but the file does not exist, i.e. the error we had before? This seems to be the right message, but I'm thinking we should future-proof it by creating a single error message source for this and the other location (L419 maybe?). Or for the same effect, perhaps we could allow this case to fall through to the original check (the way it looks like the `err == filepath.ErrBadPattern` is doing, undesirably, on master right now)?

Reply:
The same message was my explicit intention, I'll open a followup to squash both into a single place for future proofing.

Reply:
@KnVerey as promised fixes in https://github.com/kubernetes/kubernetes/pull/109504